pkg/assetfetcher/asset.go

```diff
@@ -313,12 +313,14 @@ type VulnerabilitySummary struct {
 // into an AssetEvent for downstream services.  Callers of this function are
 // expected to pass an Asset that cleanly maps to AssetEvent, meaning the
 // event must have a timestamp (because without one, it's not an event!)
+// It is guaranteed that when this function is called an asset will have a valid
```

Comment:
Nit: guaranteed is a strong word here. We could very easily change something in pkg/assetfetcher/http.go that invalidates this statement. I think it would be more correct to say something like "This function expects an asset will have a valid scanned history"

Reply:
fair point. done!